.github/workflows/ci.yml

```diff
@@ -293,9 +293,7 @@ jobs:
     - name: Building Radare2
       run: |
         # treesitter doesnt build with this because of treesitter
-        export CFLAGS="-O2 -Werror -Wno-unused-result -std=gnu89 -Wno-unicode"
```

Comment:
this is why the macos CI is failing. can you add the -Wno-unused-result flag and see what happens?, but maybe its just keeping that -Werror

```
arch/M680X/M680XDisassembler.c:2124:20: error: variable 's_cpu_type' is not needed and will not be emitted [-Werror,-Wunneeded-internal-declaration]
422

```